indico/modules/events/layout/controllers/layout.py

```diff
@@ -74,12 +75,24 @@ def _process(self):
 
 
 class RHLayoutEdit(RHLayoutBase):
-    def _process(self):
+    def _process_GET(self):
         if self.event.type_ == EventType.conference:
             return self._process_conference()
         else:
             return self._process_lecture_meeting()
 
+    def _process_POST(self):
+        old_vals = self.event.get_settings('layout')
```

Comment:
That function is not part of the indico codebase nor this PR...

Reply:
Sorry, didn't notice that was a function coming from the plugin.

Reply:
Fixed. Now this uses `layout_settings.get_all(self.event)`.